addons/editor-theme3/addon.json

```diff
@@ -328,6 +328,12 @@
         }
       ]
     },
+    {
+      "name": "Change the colors of variables and lists on the stage",
+      "id": "monitors",
+      "type": "boolean",
+      "default": false
```

Comment:
Any particular reason to default to false?

Reply:
The `affectStage` setting in editor dark mode, which also changes monitors, is off by default.

Reply:
Hmm, I wouldn't expect the "editor dark mode" addon to affect the stage by default, but having this by default seems reasonable enough.

Reply:
The editor and block colors are related though. For example, dark mode and dark blocks will usually be used together.

Reply:
@mxmou So you think this setting should stay as disabled by default?

In that case, the high-contrast presets need to set this to true... Of course, we want setting high-contrast mode from our settings page to work the same as enabling it in vanilla Scratch.
IMO, it would be simpler to enable the setting by default though

Reply:
If this setting is enabled by the default, I think the one in editor dark mode should be too.

Reply:
I would agree if Scratch had a native dark mode that also affected the stage